packages/core-data/src/entities.js

```diff
@@ -31,6 +31,8 @@ export const rootEntitiesConfig = [
 				'site_icon_url',
 				'site_logo',
 				'timezone_string',
+				'default_template_part_areas',
+				'default_template_types',
```

Comment:
Changes here also require an update to the preload path in `lib/compat/wordpress-6.8/preload.php`. See the comment above the `_fields` property.

Reply:
TIL! Thanks! fdc291d41221895d6ad6f38f8aed4fb21f9539f8

Reply:
Why didn't you keep the same order?

Reply:
See #67450.